src/pymovements/plotting/main_sequence_plot.py

```diff
@@ -21,21 +21,24 @@
 from __future__ import annotations
 
 import matplotlib.pyplot as plt
+import numpy as np
 import polars as pl
 from matplotlib.collections import Collection
+from sklearn.metrics import r2_score
```

Comment:
you need to add sklearn to the dependencies in https://github.com/aeye-lab/pymovements/blob/main/pyproject.toml